cmd/influxd/config.go

```diff
@@ -40,6 +40,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Authentication struct {
```

Comment:
I think it's worth making this is a struct -- it reads better in the code, and it's possible other params will be required in the future.